content/projects/cmints.md

```diff
@@ -0,0 +1,38 @@
+---
+title: CMintS
+repo: Manvel/cmints
+homepage: https://cmints.io/
+language:
+  - JavaScript
+license:
+  - MIT
+templates:
+  - EJS
+description: CMS and Static Content Generator created with the internationalization in mind.
+startertemplaterepo: Manvel/cmints-website
```

Comment:
This repo needs to be deployable on Netlify, so you'll need a `package.json` on the repo with a `build` command that outputs a production site. If a repo that works that way isn't available, you can always remove `startertemplaterepo` for now and add one later.

Reply:
Done, I have removed the `startertemplaterepo` for now, will see how I can make this compatible with Netlify and update accordingly later on. 
Thanks.